prow/hook/plugins.go

```diff
@@ -19,7 +19,7 @@ package hook
 // All enabled plugins. We need to empty import them like this so that they
 // will be linked into any hook binary.
 import (
-	_ "k8s.io/test-infra/prow/plugins/approve"
+	_ "k8s.io/test-infra/prow/plugins/approve" // Ensure all packages are loaded.
```

Comment:
Not sure about the utility of this change given the comment above the import block

Reply:
`golint` expects the comment to be on the same line as the first `_` aliased import. Should I remove the one before the `import` statement?